.circleci/config.yml

```diff
@@ -36,6 +36,9 @@ jobs:
           install-yarn: true
           node-version: 'latest'
       - install_android_sdk
+      - run: brew install git-lfs
+      - run: git lfs install
+      - run: git lfs pull
```

Comment:
Despite my efforts here, CircleCI is still broken. I will address this in a separate commit.